job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/publisher/PostBuildScriptsContext.groovy

```diff
@@ -6,10 +6,13 @@ import javaposse.jobdsl.dsl.DslContext
 import javaposse.jobdsl.dsl.Item
 import javaposse.jobdsl.dsl.JobManagement
 import javaposse.jobdsl.dsl.helpers.step.StepContext
+import javaposse.jobdsl.dsl.RequiresPlugin
```

Comment:
Although that import is there I'm getting the below compilation error:
```
:job-dsl-core:compileGroovy
warning: [options] bootstrap class path not set in conjunction with -source 1.6
1 warning
startup failed:
        @RequiresPlugin(id = 'postbuildscript', minimumVersion = '0.17')
        ^
/Users/victor.martinez/Workspaces/github/v1v/job-dsl-plugin/job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/publisher/PostBuildScriptsContext.groovy: 40: no jobManagement field in javaposse.jobdsl.dsl.helpers.publisher.PostBuildScriptsContext


       @RequiresPlugin(id = 'postbuildscript', minimumVersion = '0.17')
       ^
/Users/victor.martinez/Workspaces/github/v1v/job-dsl-plugin/job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/publisher/PostBuildScriptsContext.groovy: 50: no jobManagement field in javaposse.jobdsl.dsl.helpers.publisher.PostBuildScriptsContext
```

I'm struggling a bit trying to debug it, so any ideas? Did I miss anything? Probably it's something easy to spot it but I don't see what's causing that issue

Cheers